pkg/driver/node.go

```diff
@@ -283,6 +285,8 @@ func (ns *node) NodeUnstageVolume(
 	utils.TransitionVolList[volumeID] = apis.CStorVolumeAttachmentStatusUnmountUnderProgress
 	utils.TransitionVolListLock.Unlock()
 
+	logrus.Infof("Volume with ID: %v after starting unmount is in '%v' state", volumeID, utils.TransitionVolList[volumeID])
```

Comment:
Can we move this line before releasing lock? If there is any context switch happens after releasing lock and printing log, we will not get correct state printed